### PR TITLE
Update README for osquery-manager

### PR DESCRIPTION
**Pull request by james-elastic**
Update README for osquery manager.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/osquery_manager/docs/README.md [hunk 1]
```diff
@@ -1,6 +1,1513 @@
...
+      "darwin"
+    ],
+    "osquery.minutes.number": [
+      37
+    ]
+  }
+}
+```
+
+Undefined action query
```
Comment (melissaburpo) [inline packages/osquery_manager/docs/README.md:194]: @james-elastic - can we add a preface here that it's an example of an error response? Maybe even just:

> Error example: Undefined action query

File: packages/osquery_manager/docs/README.md [hunk 2]
```diff
@@ -1,6 +1,1513 @@
...
+- Run live queries against one or more agents or policies
+- View a history of the osqueries that have been run and the results of each query
+
+Osquery results are stored in Elasticsearch, so that you can use the power of the stack to search, analyze, and visualize osquery data.
+
+Once added, an Osquery page is available in Kibana under the Management section.
+
+### Supported platforms
+
+This integration supports x64_64 bit Windows, Darwin, and Linux platforms.
```
Comment (melissaburpo) [inline packages/osquery_manager/docs/README.md:17]: should this be `x86_64` instead of `x64_64` bit?

**Review by melissaburpo — APPROVED** [review]
I took a last look and saw 2 things that might need an update; but overall it looks like it's ready to go!

**Comment by james-elastic** [comment]
@melissaburpo Done

File: packages/osquery_manager/manifest.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 format_version: 1.0.0
 name: osquery_manager
 title: Osquery Manager
-version: 0.2.1
+version: 0.2.2
```
Comment (mtojek) [inline packages/osquery_manager/manifest.yml:4]: Please update the changelog file, so it's consistent with manifest.yml.

File: packages/osquery_manager/docs/README.md [hunk 4]
```diff
@@ -1,6 +1,1513 @@
...
+    "error": [
+      "action undefined"
+    ]
+  }
+}
+```
+
+
+### Exported Fields
+| Field | Description | Type |
```
Comment (mtojek) [inline packages/osquery_manager/docs/README.md:235]: Don't you prefer to use README templates? All fields/tables could be generated. Please look at the mysql integration as example: https://raw.githubusercontent.com/elastic/integrations/master/packages/mysql/_dev/build/docs/README.md

**Review by mtojek — CHANGES_REQUESTED** [review]

File: packages/osquery/changelog.yml [hunk 5]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.2.2"
```
Comment (mtojek) [inline packages/osquery/changelog.yml:2]: What about the previous version: 0.2.1? Would you mind filling the missing entry too?

**Review by mtojek — CHANGES_REQUESTED** [review]

File: packages/osquery/changelog.yml (hunk 5 above)
Comment (james-elastic) [reply packages/osquery/changelog.yml:2]: @mtojek Updated :)

**Pull request merged**